src/mudlet-mapper.xml

```diff
@@ -3111,6 +3632,61 @@ end</script>
                     <packageName></packageName>
                     <regex>^rw(?: (\w+))? (\d+)$</regex>
                 </Alias>
+                <Alias isActive="yes" isFolder="no">
+                    <name>Set exit weight</name>
+                    <script>local room = (matches[2] ~= &quot;&quot; and tonumber(matches[2]) or mmp.currentroom)
+
+local weight, exit = tonumber(matches[3]), matches[4]
+
+if not roomExists(room) then mmp.echo(&quot;Room &quot;..room..&quot; doesn't exist. It has to before we can set weights on exits.&quot;) return end
+
+setExitWeight(room, exit, weight)
+
+mmp.echo(string.format(&quot;Set the weight on the %d room going %s to %s. If it's a two-way exit, please set the reverse exit as well.&quot;, room, exit, weight))</script>
+                    <command></command>
+                    <packageName></packageName>
+                    <regex>^rwe(?: (\d+))? (\d+) (.+)$</regex>
+                </Alias>
+                <Alias isActive="yes" isFolder="no">
+                    <name>Add/delete doors</name>
+                    <script>local room = (matches[2] ~= &quot;&quot; and tonumber(matches[2]) or mmp.currentroom)
+
+local direction, status = matches[3], matches[4]
+
+if not roomExists(room) then mmp.echo(&quot;Room &quot;..room..&quot; doesn't exist. It has to before we can make doors in it.&quot;) return end
+
+local validdirs = {'e', 's', 'w', 'n', 'ne', 'se', 'sw', 'nw', 'in', 'out', 'up', 'down'}
+
+if not table.contains(validdirs, direction) then
+  mmp.echo(&quot;Can't make a door in the '&quot;..direction..&quot;' direction - available choices are:\n  &quot;..table.concat(validdirs, ', '))
+  return
+end
+
+local statusnum
+
+if status == &quot;&quot; or status == &quot;open&quot; or status == &quot;o&quot; then
+  statusnum = 1
+elseif status == &quot;closed&quot; or status ==  &quot;c&quot; then
+  statusnum = 2
+elseif status == &quot;locked&quot; or status ==  &quot;l&quot; then
+  statusnum = 3
+elseif status == &quot;clear&quot; or status == &quot;gone&quot; then
+  statusnum = 0
+end
+
+if not statusnum then mmp.echo(&quot;Unrecognized option - a door can be open, closed, locked or gone.&quot;) return end
+
+setDoor(room, direction, statusnum)
```

Comment:
Now returns `true` if a change was effected, `false` if there was no actual change (but the command was valid e.g. setting an open door when there was already a door of that type set) or `nil` + error message if there is a problem...!